packages/app/src/components/SearchBar/index.js

```diff
@@ -26,27 +26,27 @@ const SearchBar = () => {
   const [modalIsOpen, setModalIsOpen] = useState(false);
```

Comment:
I didn't change any functionality in this file - I think prettier just changed the quotes, parentheses, and the spacing.